scripts/start-setupVanillaTweaks

```diff
@@ -54,23 +56,23 @@ if [[ "$VANILLATWEAKS_FILE" ]]; then
 fi
 
 # Download and unzip packs
-if [[ "$PACKS" ]] && [[ "$VT_VERSION" ]]; then
+if [[ "$PACKS" ]] && [[ "$PACKS" != "null"]] && [[ "$VT_VERSION" ]]; then
```

Comment:
I'm not sure offhand why test revealed a bash error here. The syntax seems to be defensive. I'll help investigate this too.

Reply:
We can also use //empty in the jq like I did for the forge API. Then it'll return empty array like we'd expect. 🤔

Reply:
It just dawned on me that the failing test is the full startup one that only sets EULA and VERSION. On master it's 

https://github.com/itzg/docker-minecraft-server/blob/de0188de3d7bcd919216ed20a04891896c347e6e/tests/fulltests/vanilla-latest/docker-compose.yml#L12

Now I'm wondering why the VANILLATWEAKS logic is even getting hit here.